model/table_column.go

```diff
@@ -217,6 +217,15 @@ func (t *tablecol) AutoUpdate() string {
 	return t.autoUpdate.Value
 }
 
+func (t *tablecol) SetEnumValues(enumValues []string) TableColumn {
+	t.enumValues = enumValues
+	return t
+}
+
+func (t *tablecol) EnumValues() []string {
```

Comment:
Please use the pattern used in other accessors (return `chan Type`). If you feel that that is overkill, that's understandable, but if that were to be changed, all of those accessors should be changed. We don't want just one of the APIs acting differently.

Reply:
fixed